### PR TITLE
code review request

### PR DESCRIPTION
**Pull request by StevenHowlett**
please check logic, naming and formatting
@Christgo
@Mackc2

### CODE REVIEW

File: prac5/state_names.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
```
Comment (Christgo) [inline prac5/state_names.py]: You can remove this line if you'd like

File: prac5/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+longest_word_length = 0
+for word in words:
+    if word not in word_to_count:
+        word_to_count[word] = 1
+    else:
+        word_to_count[word] += 1
+    if len(word) > longest_word_length:
+        longest_word_length = len(word)
+for word,count in sorted(word_to_count.items()):
+    print("{:<{}} : {}".format(word,longest_word_length,count))
```
Comment (Christgo) [inline prac5/word_occurrences.py:13]: "<" may not be necessary in your code

File: prac5/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+word_to_count = {}
+user_text = input('text')
+words = user_text.split()
+longest_word_length = 0
+for word in words:
+    if word not in word_to_count:
+        word_to_count[word] = 1
+    else:
+        word_to_count[word] += 1
```
Comment (Christgo) [inline prac5/word_occurrences.py:9]: A return (enter) may help improve readability of the program

**Review by Christgo — COMMENTED** [review]
Add a return and remove "<"

File: prac5/state_names.py (hunk 1 above)
Comment (StevenHowlett) [reply prac5/state_names.py]: thanks for that I removed that comment :)

File: prac5/word_occurrences.py (hunk 2 above)
Comment (StevenHowlett) [reply prac5/word_occurrences.py:13]: thanks i did not realize that was a default

File: prac5/word_occurrences.py (hunk 3 above)
Comment (StevenHowlett) [reply prac5/word_occurrences.py:9]: thanks read ability improved

File: prac5/state_names.py (hunk 1 above)
Comment (Mackc2) [inline prac5/state_names.py]: Redundant Comment should be removed

**Pull request merged**